climada/hazard/tc_tracks.py

```diff
@@ -1383,7 +1383,7 @@ def write_hdf5(self, file_name, complevel=5):
         df_attrs = pd.DataFrame([t.attrs for t in data], index=ds_combined["storm"].to_series())
         ds_combined = xr.merge([ds_combined, df_attrs.to_xarray()])
 
-        encoding = {v: dict(zlib=True, complevel=complevel) for v in ds_combined.data_vars}
+        encoding = {v: dict(zlib=False, complevel=complevel) for v in ds_combined.data_vars}
```

Comment:
Great! However, this turns off compression for all variables and might drastically increase the file size. We want to apply zlib to numerical data, so something like the following might be appropriate:

```python
def zlib(var):
    if np.issubdtype(var.dtype, float) or np.issubdtype(var.dtype, int):
        return True
    return False

encoding = {v: dict(zlib=zlib(v), complevel=complevel) for v in ds_combined.data_vars}
```

Reply:
Good point. I added such a function.

I also made the (priviously private) function that connects the now split modules a public function, so no private function is imported over modules. Any other comments?